R/mainTests.r

```diff
@@ -1120,12 +1109,15 @@ overcompleted_cols <- function(data, meta) {
 
   regional_matrix <- matrix(geography_matrix[2, ], nrow = 1)
   la_matrix <- matrix(geography_matrix[3, ], nrow = 1)
+  lsip_matrix <- matrix(geography_matrix[7, ], nrow = 1)
 
   overcomplete_geographies <- c(
     unlist(apply(regional_matrix, 1, overcomplete_regional_cols)),
     unlist(apply(la_matrix, 1, overcomplete_la_cols)),
-    unlist(apply(geography_matrix[4:12, ], 1, overcomplete_mid_cols)),
-    unlist(apply(geography_matrix[13:16, ], 1, overcomplete_low_cols))
+    unlist(apply(geography_matrix[4:6, ], 1, overcomplete_mid_cols)),
+    unlist(apply(lsip_matrix, 1, overcomplete_lsip_cols)),
+    unlist(apply(geography_matrix[8:13, ], 1, overcomplete_mid_cols)),
```

Comment:
Could line 1117 be "...matrix[c(4:6, 8:13), ] ...." and remove the need for line 1119?

Reply:
Good point, done that.